index.html

```diff
@@ -5,9 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <title>My Portfolio</title>
 </head>
 <body>
-    
+    <div class="header">
+        <a href="#"><img src="/image/images.png" width="100px"></a>
+        <nav> 
+        <a href="#" class="fa fa-bars"></a>
+        </nav>
+    </div>
```

Comment:
It would be nice if we enclose the section in a  `header` tag and not `div`. 🤗

- Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 🛠 👍

Your design
![image](https://github.com/Nomarg/My-Portfolio/assets/78251927/e41fe2eb-fefb-4ae2-b479-99b5c5c73980)

Figma
![image](https://github.com/Nomarg/My-Portfolio/assets/78251927/71bc0955-677d-405a-88f1-26c924dadb87)

Please kindly ensure that the section has the margins (top, bottom, left, right) defined in the design.

* Kindly follow the original vision, if it's pretty and works, it will be a wonderful website. 👍🛠